app/controllers/organizations_controller.rb

```diff
@@ -0,0 +1,43 @@
+class OrganizationsController < ApplicationController
+  before_filter :authenticate_user!, only: [:create, :new, :edit, :update]
+  before_action :load_presented_record, only: [:show, :edit]
```

Comment:
Would you add empty `show` and `edit` methods [like the ones here](https://github.com/MAPC/developmentdatabase/blob/edit/app/controllers/developments_controller.rb#L8) to indicate that those methods do occur?